### PR TITLE
Claim card design update

### PR DESCRIPTION
**Pull request by samcoforma**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- _(Summarize the changes that have been made to the platform)_
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/64214
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    'Step 3 of 5: Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Step 4 of 5: Preparation for notification',
+  COMPLETE: 'Step 5 of 5: Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
+}
+
+const formatDate = date => moment(date).format('MMMM D, YYYY');
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:20]: I would swap`moment` for `date-fns` here

Comment (jerekshoe) [reply src/applications/claims-status/components/ClaimsListItemV3.jsx:20]: I would take a look at the `buildDateFormatter` function in the helpers file

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+const getTitle = claim => {
+  return `Claim for ${getClaimType(claim)}`;
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:23]: I would add a `.toLowerCase()` at the end of the call to `getClaimType`

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        {humanStatus && <p>{humanStatus}</p>}
+        <p>{getLastUpdated(claim)}</p>
+      </div>
+      {inProgress && documentsNeeded ? (
+        <va-alert
+          close-btn-aria-label="Close notification"
+          disable-analytics="false"
+          full-width="false"
+          status="warning"
+          visible="true"
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:99]: You don't need this prop

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+      </ul>
+      <div className="card-status">
+        {humanStatus && <p>{humanStatus}</p>}
+        <p>{getLastUpdated(claim)}</p>
+      </div>
+      {inProgress && documentsNeeded ? (
+        <va-alert
+          close-btn-aria-label="Close notification"
+          disable-analytics="false"
+          full-width="false"
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:97]: You don't need these props either

File: src/applications/claims-status/sass/claims-status.scss [hunk 5]
```diff
@@ -135,6 +150,42 @@
...
+
+h3 .usa-label {
+  margin-top: 0;
+  margin-bottom: 1em;
+}
+
+// Index page V3
+.claim-list-item-container .card-status {
+  margin: 0 0 1em;
+  // display: flex;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:180]: I'm assuming this isn't needed?

File: src/applications/claims-status/components/StemClaimListItemV3.jsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+import recordEvent from 'platform/monitoring/record-event';
+
+export default function StemClaimListItemV3({ claim }) {
+  if (!claim.attributes.automatedDenial) {
+    return null;
+  }
+  const formattedDeniedAtDate = moment(claim.attributes.deniedAt).format(
```
Comment (jerekshoe) [inline src/applications/claims-status/components/StemClaimListItemV3.jsx:10]: Similar thing here in regards to `moment` vs `date-fns`

**Comment by jerekshoe** [comment]
I think in progress cards are supposed to have some sort of drop shadow aren't they?

File: src/applications/claims-status/sass/claims-status.scss [hunk 7]
```diff
@@ -97,18 +103,28 @@
...
+}
+
 .communications,
 .status {
-  margin: 0 0 0.5em;
+  margin: 0;
+}
+
+.communications li {
+  line-height: 2;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:123]: I think this should be 1.5 (the default)

File: src/applications/claims-status/sass/claims-status.scss [hunk 8]
```diff
@@ -135,6 +151,42 @@
...
+  font-style: normal;
+  font-weight: normal;
+  line-height: 1.5;
+  margin-bottom: 0.5em;
+  margin-top: 0.5em;
+}
+
+.submitted-on {
+  line-height: 1.5;
+  font-size: 16px;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:169]: I don't think we need either of these, as these are the defaults

File: src/applications/claims-status/sass/claims-status.scss [hunk 9]
```diff
@@ -135,6 +151,42 @@
...
+h3 .usa-label {
+  display: block;
+  width: fit-content;
+  font-size: 1.6rem;
+  font-family: var(--font-source-sans);
+  font-style: normal;
+  font-weight: normal;
+  line-height: 1.5;
+  margin-bottom: 0.5em;
+  margin-top: 0.5em;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:164]: I think it's safe to remove these. It seems like it makes more sense to put the margin values in the specific selectors (i.e. `.submitted-on` and `h3 .usa-label`)

File: src/applications/claims-status/sass/claims-status.scss [hunk 10]
```diff
@@ -135,6 +151,42 @@
...
+
+// Index page V3
+.claim-list-item-container .card-status {
+  margin: 0 0 1em;
+  // display: flex;
+
+  p {
+    margin-top: 0;
+    margin-bottom: 0;
+    line-height: 1.5;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:186]: This is the default already

File: src/applications/claims-status/sass/claims-status.scss [hunk 11]
```diff
@@ -170,6 +222,13 @@
   white-space: pre-line;
 }
 
+.claim-list-item-header {
+  margin-top: 0;
+  margin-top: -1.5rem;
+  white-space: pre-line;
+  line-height: 1.44;
```
Comment (jerekshoe) [inline src/applications/claims-status/sass/claims-status.scss:229]: I don't think we need any of these

File: src/applications/claims-status/components/ClaimsListItemV3.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/claims-status/components/ClaimsListItemV3.jsx:20]: This line would be `const formatDate = buildDateFormatter(DATE_FORMATS.LONG_DATE);`

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+          status="warning"
+          visible="true"
+          slim
+          uswds
+        >
+          <div className="vads-u-margin--0">
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:103]: Do we need this class here?

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+    claim.attributes.claimPhaseDates?.phaseChangeDate,
+  );
+
+  return `Last updated: ${updatedOn}`;
+};
+
+const isClaimComplete = claim => {
+  const { decisionLetterSent, status } = claim.attributes;
+
+  return decisionLetterSent || status === 'COMPLETE';
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:37]: One thing to keep in mind here is this doesn't just return true when the claim is closed, so we may want to reconsider the logic

File: src/applications/claims-status/containers/YourClaimsPageV2.jsx [hunk 14]
```diff
@@ -130,20 +137,33 @@
...
+          : StemClaimListItemV2;
       return <StemClaimListItem key={claim.id} claim={claim} />;
     }
 
     // START lighthouse_migration
-    const { useLighthouse } = this.props;
-    const ClaimsListItem = useLighthouse
-      ? ClaimsListItemLighthouse
-      : ClaimsListItemEVSS;
+    let ClaimsListItem = <></>;
```
[the remark this answers is not in the document]
Comment (jerekshoe) [reply src/applications/claims-status/containers/YourClaimsPageV2.jsx:159]: @kjduensing you think we should fix this warning?

**Review by jerekshoe — APPROVED** [review]

**Comment by jerekshoe** [comment]
@department-of-veterans-affairs/va-platform-cop-frontend can you review the changes to `featureFlagNames.jsx`?

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**